imports/plugins/core/email/client/components/emailLogs.js

```diff
@@ -0,0 +1,132 @@
+import React, { Component, PropTypes } from "react";
+import moment from "moment";
+import { Card, CardHeader, CardBody, CardGroup, Icon, Loading } from "/imports/plugins/core/ui/client/components";
+import MeteorGriddle from "/imports/plugins/core/ui-grid/client/griddle";
+import { Jobs } from "/lib/collections";
+import { i18next } from "/client/api";
+import "./emailConfig.css";
+
+
+class EmailLogs extends Component {
+
+  renderEmailsTable() {
+    const filteredFields = ["data.to", "updated", "data.subject", "status"];
+    const filteredFieldsColumns = ["data.to", "updated", "data.subject", "status"];
+    const noDataMessage = i18next.t("admin.logs.noEmails");
+
+
+    // helper adds a class to every grid row
+    const customRowMetaData = {
+      bodyCssClassName: () =>  {
+        return "email-grid-row";
+      }
+    };
+
+
+    const ColumnData = React.createClass({
```

Comment:
and also, don't create a class in a class. That needs to be moved to its own file.